### DISCUSSION TITLE
Kernel Clean is showing different kernel version

### DISCUSSION THREAD

**Discussion by ChristophCaina** [tteck/Proxmox#629] [Help Needed] [answered]
Hi, I just found your scripts and decided to try some of them...
But now, I immediately run into an issue which I do not really understand:

Based on the information on my Proxmox Host, I am running Kernel 5.15.60-2.pve
![grafik](https://user-images.githubusercontent.com/26391061/195276140-4aead882-e6b5-4741-9ec8-12499639f396.png)

But when I'm running the KernelClean script, it is showing an pretty old kernel version as "active / latest kernel"
![grafik](https://user-images.githubusercontent.com/26391061/195276269-2455e78e-12cc-4667-9160-40b25f53a96e.png)

the active one - and many more newer kernels are listed as "can be removed":
![grafik](https://user-images.githubusercontent.com/26391061/195276426-b287c6fe-afcc-4950-a101-77ca1f2201c4.png)

Is there anything I can do to verify which kernel is active and running?
![grafik](https://user-images.githubusercontent.com/26391061/195276929-88609126-7d9d-4eb3-8f64-60392c89899b.png)
![grafik](https://user-images.githubusercontent.com/26391061/195277766-a05937c0-7deb-42d3-bea9-0277e06574f3.png)

**Comment by tteck** [comment, marked as answer]
Whatever `pveversion` says is what you go by.

**Reply by ChristophCaina** [reply]
so, I can "safely" delete all the kernels that are listed in the script?
Thanks :)

**Reply by tteck** [reply]
yes
